SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -4,14 +4,22 @@ Volume {
 
 	var <volume = 0.0, <lag = 0.1, <isMuted = false;
 
-	var <ampSynth, <numOutputChannels, defName, updateFunc, initFunc;
+	var <ampSynth, <numOutputChannels, <defName, updateFunc, initFunc;
 	var <>window;
+	var <group, baseAmpSynth;
```

Comment:
"base amp" sounds like the amplitude of the base. 

It is a little unclear what the difference between `ampSynth` and `baseAmpSynth` is...

Reply:
baseAmpSynth should maybe be called basicAmpSynth or permAmpSynth:
It is only made once, and sends its newMsg to the server when needed. 
ampSynth is either nil when baseAmpSynth is not running, 
or is assigned to baseAmpSynth when running. 

could also be a flag ampSynthRunning, would that be clearer?

Reply:
@adcxyz Ah now I understand, `ampSynth` becomes `baseAmpSynth` when the volume is running. So I suppose it would be possible to have only `ampSynth` and keep it permanent? Then you'd need the flag `ampSynthRunning`, right? But you could register the permanent amp synth and then you'd know it it isn't running, no?

Reply:
yes, that is the idea. 
The next steps are:
- the ampSynth gets a SendReply to send its current settings back to all clients; 
- the clients update their volume settings with the incoming info
- because all clients use the same volume group/nodeID, 
any client can start a volume synth which is accessible equally from all clients.